cdap-app-fabric/src/main/java/io/cdap/cdap/internal/profile/ProfileService.java

```diff
@@ -17,11 +17,13 @@
 package io.cdap.cdap.internal.profile;
 
 import com.google.common.annotations.VisibleForTesting;
+import com.sun.tools.internal.jxc.ap.Const;
```

Comment:
Wrong import? We shouldn't be using this class.

Reply:
Removed.